### DISCUSSION TITLE
Welcome to ml2scratch Discussions!

### DISCUSSION THREAD

**Discussion by champierre** [champierre/ml2scratch#23] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Jagui5** [comment]
Hello, 

I've used stretch3.github.io with ml2scratch and the pencil extension in a workshop for teachers that had mostly students (11-15 years old), the purpose was to teach the very basics of IA in a practical way using Scratch that the kids already learn in school,
in our case we trained the labels by writing numbers in japanese.

The attendants have loved it !

Thank you very much the team for developping this tool.

Greetings from France,

Javier

**Reply by champierre** [reply]
Thank you so much for your report. I am Japanese, but grew up in Neuilly-sur-Seine near to Paris when I am very small, so I have special feeling to France. I am very glad that my product is used in my second home country.

**Reply by Jagui5** [reply]
I will go soon visit that area, a friend just moved closed there (Paris 17 ème arrondissement).
I did a little overview of the workshop, it's in French, but it's very visual.
https://youtu.be/Xem5Ha49NjA?si=D1KpoBqXi4Gdg-7r
